aws/load-balancer.tf

```diff
@@ -60,6 +60,51 @@ resource "aws_s3_bucket" "rancher_lb_access_logs" {
   tags = "${merge(map("Name", "${var.rancher_lb_access_logs_bucket}"), var.cloud_tags)}"
 }
 
+data "aws_elb_service_account" "default" {}
+
+data "aws_caller_identity" "default" {}
+
+resource "aws_s3_bucket_policy" "rancher_lb_access_logs" {
+  bucket = "${aws_s3_bucket.rancher_lb_access_logs.bucket}"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "${data.aws_elb_service_account.default.arn}"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/AWSLogs/${data.aws_caller_identity.default.account_id}/*"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "delivery.logs.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/AWSLogs/${data.aws_caller_identity.default.account_id}/*",
```

Comment:
Best to simplify and change to `arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/*`

Reply:
I'll accept as is